wiki/osu!dev_Discord_server/de.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+  - development
+  - Softwareentwicklung
+---
+
+# osu!dev Discord-Server
+
+**osu!dev** ist der offizielle [Discord](https://discordapp.com)-Server, um an den Open-Source und Community-Projekten von osu! zu arbeiten. Das ist der Marktplatz, um mit dem [osu!-Team](/wiki/People/The_Team) in Kontakt zu treten und an der Entwicklung teilzunehmen.
+
+Der dauerhafte Einladungslink lautet <https://discord.gg/ppy>.
+
+## Verifizierung
+
+Neue Mitglieder des Servers haben normalerweise nicht die Erlaubnis Nachrichten zu senden. Im `#arrival`-Kanal kann man sich mit einem kurzen Überblick über seine Intentionen vorstellen. Ein Mitglied des osu!-Teams wird dann eine Rolle hinzufügen, um die Beschränkung aufzuheben.
+
+## Projekte
+
+Jedes Projekt wird in seinem dafür vorgesehenen Kanal diskutiert:
+
+| Projekt | Kanal |
+| :-- | :-- |
+| [osu!lazer](https://github.com/ppy/osu) | `#osu-lazer` |
+| [osu!framework](https://github.com/ppy/osu-framework) | `#osu-framework` |
+| [osu!web](https://github.com/ppy/osu-web) | `#osu-web` |
+| [osu!](/wiki/Game_mode/osu!) [Star Rating](/wiki/Beatmapping/Star_rating) und [Performancepunkte](/wiki/Performance_points) | `#difficulty-osu` |
+| [osu!taiko](/wiki/Game_mode/osu!taiko) Star Rating und Performancepunkte | `#difficulty-taiko` |
+| [osu!catch](/wiki/Game_mode/osu!catch) Star Rating und Performancepunkte | `#difficulty-catch` |
+| [osu!mania](/wiki/Game_mode/osu!mania) Star Rating und Performancepunkte | `#difficulty-mania` |
+| [Beatmap Spotlights](/wiki/Beatmap_Spotlights) | `#osu-spotlights` |
+| [osu! wiki](https://github.com/ppy/osu-wiki) | `#osu-wiki` |
+| [Project Loved](/wiki/Project_Loved) | `#osu-loved` |
+| [Modding](/wiki/Modding) Diskussionen und [NAT Meetings](/wiki/Modding/NAT_meetings) | `#modding` |
+| [Mapper Guilde](/wiki/Mappers_Guild) | `#mappers-guild` |
```

Comment:
`Mapper's Guild` is a name, so I don't think it should be translated(just like `Project Loved`). Though if you do want to translate it, `Guilde` is written as `Gilde`.
```suggestion
| [Mapper's Guild](/wiki/Mappers_Guild) | `#mappers-guild` |
```

Reply:
You're right. We should take the official name. Especially with such words, I still have problems sometimes, because I'm not sure if I should translate them. Also couldn't find usages of that term elsewhere on the German wiki. **Translating is hard**.